vector/src/main/java/im/vector/app/features/home/room/detail/composer/VoiceMessageHelper.kt

```diff
@@ -76,7 +76,7 @@ class VoiceMessageHelper @Inject constructor(
         startRecordingAmplitudes()
     }
 
-    fun stopRecording(convertForSending: Boolean): MultiPickerAudioType? {
+    suspend fun stopRecordingAndGetAudio(convertForSending: Boolean): MultiPickerAudioType? {
```

Comment:
I renamed these functions as the name `stopRecording` doesn't imply a return value. It also becomes more confusing with its original name as to why it needs to be a suspend functions, but I think this makes it clearer